src/interface/src/app/treatments/expanded-stand-data-chart/expanded-stand-data-chart.component.scss

```diff
@@ -0,0 +1,16 @@
+@import "mixins";
+
+.expanded-stand-data {
+  padding: 20px;
+  height: 70vh;
+}
+
+::ng-deep {
+  sg-modal.stand-data-modal {
```

Comment:
@lastminutediorama I couldn't find this exact variant on the modal component. We have `shortHeader` but this one is a bit bigger according to figma, and font styles a tad different.  Not sure if we should just add one more thing or audit the headers and see if we need to refactor.
Also the close icon is different in figma for these "expanded" panels... maybe its just another, very similar, component?

Reply:
I'm not sure either. It's probably worth considering a refactor of the headers.

Reply:
But to be clear, I'm okay with whatever you want to do for this PR.